lib/mayaUsd/fileio/translators/translatorXformable.cpp

```diff
@@ -378,14 +378,16 @@ static bool _pushUSDXformToMayaXform(
     const UsdMayaPrimReaderArgs&    args,
     const UsdMayaPrimReaderContext* context)
 {
+    MTime::Unit timeUnit = MTime::uiUnit();
+    double timeSampleMultiplier = (context != nullptr) ? context->GetTimeSampleMultiplier() : 1.0;
+
```

Comment:
Re-ordered the lines to be the same as in the other function (_pushUSDXformOpToMayaXform).